### PR TITLE
Add limitation related to compileSdkVersion

### PR DESCRIPTION
**Pull request by TWiStErRob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Paparazzi does not currently support namespaced resources. If you are seeing thi
 not present in the project's `gradle.properties`:
 `android.nonTransitiveRClass=true`
 
--------- 
+--------
```
Comment (jrodbx) [inline README.md:160]:
```suggestion
-------
```

File: README.md [hunk 2]
```diff
@@ -168,6 +168,24 @@ java.lang.NullPointerException
 ```
 Running tests from the IDE requires Android Studio Arctic Fox or later. 
 
+--------
```
Comment (jrodbx) [inline README.md:171]:
```suggestion
-------
```

File: README.md [hunk 3]
```diff
@@ -168,6 +168,24 @@ java.lang.NullPointerException
...
+#### Could not find ... resource matching value 0x... (resolved name: ...) in current configuration.
+Ex:
+```
+Could not find dimen resource matching value 0x10500C0 (resolved name: config_scrollbarSize) in current configuration.
+android.content.res.Resources$NotFoundException: Could not find dimen resource matching value 0x10500C0 (resolved name: config_scrollbarSize) in current configuration.
+
+Could not find integer resource matching value 0x10E00B4 (resolved name: config_screenshotChordKeyTimeout) in current configuration.
+android.content.res.Resources$NotFoundException: Could not find integer resource matching value 0x10E00B4 (resolved name: config_screenshotChordKeyTimeout) in current configuration.
+```
+`compileSdkVersion` has to be 29 or higher. This is a safe and independent change from `minSdkVersion` and `targetSdkVersion`, those can stay as is.
```
Comment (jrodbx) [inline README.md:182]:
```suggestion
`compileSdkVersion` has to be 29 or higher. 
```

File: README.md [hunk 4]
```diff
@@ -168,6 +168,24 @@ java.lang.NullPointerException
...
+```kotlin
+android {
+	compileSdkVersion(29)
```
Comment (jrodbx) [inline README.md:185]:
```suggestion
  compileSdkVersion 29
```

**Review by jrodbx — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (TWiStErRob) [reply README.md]: Not a fan of Kotlin DSL?

**Pull request merged**